improver_tests/wxcode/wxcode/test_utilities.py

```diff
@@ -256,37 +260,17 @@ def tearDown(self):
         os.remove(self.nc_file)
         os.rmdir(self.data_directory)
 
-    def test_basic(self):
-        """Test that the function returns a cube."""
-        result = add_wxcode_metadata(self.cube)
-        self.assertIsInstance(result, Cube)
-
-    def test_metadata_set(self):
-        """Test that the metadata is set correctly."""
-        result = add_wxcode_metadata(self.cube)
-        self.assertEqual(result.name(), 'weather_code')
-        self.assertEqual(result.units, Unit("1"))
-        self.assertArrayEqual(result.attributes['weather_code'], self.wxcode)
-        self.assertEqual(result.attributes['weather_code_meaning'],
-                         self.wxmeaning)
-
-    def test_metadata_removes_bounds(self):
```

Comment:
You have moved this functionality into weather_code.py but it is not tested there. Any particular reason?

Reply:
Laziness, combined with a very rational fear of the `test_WeatherSymbols` module.  I'll look at this again!

Reply:
My response to this has included using the new cube setup utility to make the input cube.  I have only done this for this one specific test class, since the old function is sourced in a lot of places.  But this is something that could / should be updated everywhere in this file (it's one of the last modules under the old old "update unit tests" ticket).

This PR or a separate one?

Reply:
Separate PR. I'm just running the tests and then I am happy for this PR to be merged.